pkg/kubernetes/pod/labels.go

```diff
@@ -39,7 +39,7 @@ func (check ProxyUUIDLabelCheck) Run() error {
 
 // Suggestion implements common.Runnable
 func (check ProxyUUIDLabelCheck) Suggestion() string {
-	panic("implement me")
+	return fmt.Sprintf("Verify that the pod is in a meshed namespace. Try: \"osm namespace list\"")
```

Comment:
do we anticipate having customers who won't be using the osm cli?

Reply:
I think it is reasonable to point customers to `osm` CLI...